src/test/controllers/survey.controller.test.js

```diff
@@ -0,0 +1,156 @@
+import mockAnswerData from '../fixtures/answerA001_1';
+import mockSurveyData_mentee from '../fixtures/surveyA001_1';
+import mockSurveyData_mentor from '../fixtures/surveyB001_1';
+import mongoose from 'mongoose';
+import should from 'should';
+import surveyCallback from '../../server/config/json/survey.callback';
+import rp from 'request-promise';
+
+/*
+ * Test for Survey API
+ */
+
+const Survey = mongoose.model('survey');
+
+const API_BASE_URL = 'http://localhost:8000/survey';
+
+describe('Test Survey API', function () {
+  describe('/create', function () {
+    it(': Save mentee survey.', function (done) {
+      let options = {
+        method: 'POST',
+        uri: API_BASE_URL + '/create',
+        form: mockSurveyData_mentee.surveyA001_1,
+        resolveWithFullResponse: true,
+        json: true,
+      };
+
+      rp(options)
+        .then((result) => {
+          result.statusCode.should.equal(200);
+          result.body.survey_id.should.equal(mockSurveyData_mentee.surveyA001_1.survey_id);
```

Comment:
Yes, it need to be shorten. will change mock data as
`mockSurveyData_mentee => menteeData`
`mockSurveyData_mentee => mentorData`
like this!